packages/fireproof/tsconfig.json

```diff
@@ -15,6 +15,9 @@
     "declaration": true,
     "outDir": "dist",
     "rootDir": "src",
-    "forceConsistentCasingInFileNames": true
+    "forceConsistentCasingInFileNames": true,
+    "paths": {
+      "@fireproof/encrypted-blockstore/*": ["./node_modules/@fireproof/encrypted-blockstore/dist/*"]
+    }
```

Comment:
Added `paths` here so we can get rid of those `@ts-ignore` comments in the `eb-*.ts` files when importing from the multiple entry points.

Reply:
Decided to split out this little tidbit into https://github.com/fireproof-storage/fireproof/pull/66

Reply:
Nvm -- brought this back in